### ISSUE TITLE
Empty output file content means error or nothing?

### ISSUE THREAD

**Issue by Chensanyu** [taylor-lab/hotspots#13]
Hello, 
    Thank you very much for providing such a useful software. However, when I run my own data, the output result file has nothing but the column name. Empty results indicate that there are no hotspots or errors in operation.

**Comment by kpjonsson** [comment]
Please provide a reproducible example. Or at least give me an idea what your input data consists of.

**Comment by Chensanyu** [comment]
Hello,
Thank you very much for your reply. I ran two copies of my own data, one named 'original' and one named 'modified'.
When I ran the ‘original’ file, I encountered ‘**Error in rep(1, aa.length - length(pval)) : invalid 'times' argument  Calls: lapply -> lapply -> FUN**’. The error I found seems to be inconsistent with your answer to issue1 from kgaonkar6 ,   so I deleted the genes that caused the error.  'Modified' is the file that deletes the genes that caused the error. I successfully ran the 'Modified'  file, but the output result file was empty except for the column name.
**‘original’ and 'Modified'are in the zip file.**
**Sincerely hope to get your answer about the problems encountered by these two files.**
System environment: **Ubuntu 16.04.4**
[files.zip](https://github.com/taylor-lab/hotspots/files/3107969/files.zip)
